pkg/role/sdkserviceapi.go

```diff
@@ -91,6 +91,19 @@ var (
 				},
 			},
 		},
+
+		// system:public role is used for any unauthenticated user.
+		// They can only use standard volume lifecycle commands.
+		"system.public": {
```

Comment:
Add support for `!` not allowed RBAC. `!*inspect*`. Remove `!` then send the string to `match(r[1:], s)`

```		"system.public": {
			&api.SdkRule{
				Services: []string{"volume"},
				Apis:     []string{"!create", "*"},
			},
			&api.SdkRule{
				Services: []string{"mountattach", "cloudbackup", "migrate"},
				Apis:     []string{"*"},
			},
			&api.SdkRule{
				Services: []string{"identity"},
				Apis:     []string{"version"},
			},
		},
```

Update the documentation in the Role API in the api.proto to support this, and show examples.

Reply:
This will happen in a separate PR.